### PR TITLE
Correct formatting and style

### PR DESCRIPTION
**Pull request by shannonxtreme**
Page: /docs/tasks/configmap-secret/managing-secret-using-kubectl/

## Description

Made minor changes to content for consistency and clarity. Corrected minor grammar errors. 

/sig docs
/kind cleanup
/language-en

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

File: content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md [hunk 1]
```diff
@@ -54,17 +53,18 @@ kubectl create secret generic db-user-pass \
   --from-file=password=./password.txt
 ```
 
-You do not need to escape special characters in passwords from files
-(`--from-file`).
+You do not need to escape special characters in passwords from files that you
```
Comment (kbhawkey) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md:56]: Hi @shannonxtreme . The changes look good. Thanks.
nit:  What do you think about possibly rephrasing line 56 (the important bit is: special characters in the password string do not need to be escaped). Optionally, add the sentence as a note.
Something like:
When including a password from a file, you do not need to escape [the] special characters in the password string.

Comment (shannonxtreme) [reply content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md]: This stumped me more than it should have, lol. I reworded to "You do not need to escape special characters in password strings that you include in a file." Let me know what you think.

Comment (kbhawkey) [reply content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md]: Sure. The original text was clumsy. Thanks!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27737--kubernetes-io-master-staging.netlify.app/docs/tasks/configmap-secret/managing-secret-using-kubectl/#create-a-secret

/approve

**Pull request merged**